app/assets/javascripts/plot/highchartsops.coffee

```diff
@@ -92,7 +92,7 @@ class window.HighchartsOps extends PlotOps
     super(resize, reset, registerPen, resetPen, addPoint, updatePenMode, updatePenColor)
     @_chart              = Highcharts.chart(elemID, {})
     @_penNameToSeriesNum = {}
-
+    @_chart.options.exporting.buttons.contextButton.menuItems.pop()
```

Comment:
@TheBizzle I thought it might be useful for users to have access to exporting the data to XLS but that might be a bit redundant. 

The export-csv plugin for highcharts adds three functions to the highchart menu

1. Download CSV - self-explanatory
2. Download XLS - self-explanatory
3. Show table - basically just displays an HTML table of the data, this is the most useless because it ruins the formatting and just gets in the way.

I'll add an extra pop to get ride of the Download XLS so we only have the CSV functionality.